src/components/dashboard/purchase-card.svelte

```diff
@@ -24,13 +25,12 @@
         {purchase.product.name}
         <span class="type">{purchase.product.type || ''}</span>
       </div>
-      <time datetime={purchase.time} title={new Date(purchase.time).toLocaleString('ru')}>
-        {new Date(purchase.time).toLocaleString('ru', {month: '2-digit', day: '2-digit'})}
-      </time>
+      <time datetime={purchase.time}>{formatTime(purchase.time)}</time>
```

Comment:
Why did you remove the `title` attribute? It can be helpful to know the exact time when hovering over it

Reply:
This representation is already as exact as it gets (sans seconds, but who gives a shit)

Reply:
ah, ok. Previously, it was only month and day, but I didn't check `formatTime` after this change